api/v1alpha1/workflownode_types.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:shortName=wfn
+type WorkflowNode struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the behavior of a pod chaos experiment
+	Spec WorkflowNodeSpec `json:"spec"`
+
+	// +optional
+	// Most recently observed status of the chaos experiment about pods
+	Status WorkflowNodeStatus `json:"status"`
+}
+
+type WorkflowNodeSpec struct {
+	TemplateName string       `json:"template_name"`
+	WorkflowName string       `json:"workflow_name"`
```

Comment:
What does `WorkflowName` mean here? Is it the workflow the node belongs to?

Reply:
> Is it the workflow the node belongs to?
Yes.